src/test/java/tech/ydb/io/r2dbc/statement/YdbDMLStatementTest.java

```diff
@@ -120,7 +120,7 @@ public void bindNullParameterTest() {
 
     @Test
     public void bindNullYdbParameterTest() {
-        YdbQuery query = new YdbQuery("test_sql", List.of("$testParamA", "$testParamB"), QueryType.DML);
+        YdbQuery query = new YdbQuery("test_sql", List.of("$testParamA", "$testParamB"), QueryType.DML, List.of());
```

Comment:
YdbQuery has two constructors YdbQuery("test_sql", List.of("$testParamA", "$testParamB"), QueryType.DML) { this( .., List.of() ) }